git/githistory/rewriter.go

```diff
@@ -188,6 +193,15 @@ func (r *Rewriter) Rewrite(opt *RewriteOptions) ([]byte, error) {
 		vPerc = perc
 	}
 
+	var objectMapFile *os.File
+	if len(opt.ObjectMapFilePath) > 0 {
+		objectMapFile, err = os.Create(opt.ObjectMapFilePath)
```

Comment:
Should we pass the `O_EXCL` mode here? Doing so would cause the program to error when a file already exists in that location, instead of writing over it.

```go
f, err := os.OpenFile(opt.ObjectMapFilePath, os.O_RDWR|os.O_CREATE|os.O_EXCL, 0666)
if err != nil {
        // ..
}
```

Reply:
Yes, I think that's safer.